### PR TITLE
Adding Fariba to the team page

### PR DESCRIPTION
**Pull request by frb-yousefi**

### CODE REVIEW

**Comment by bobturneruk** [comment]
Hi @frb-yousefi - great to see this! If it's ready for review, please select one or more reviewers (top right of interface).

**Review by drj11 — APPROVED** [review]
Great. Please merge and delete the branch.

**Comment by bobturneruk** [comment]
Hi @frb-yousefi - it's OK to merge with one approving review. This may not be clear in the readme etc. Please let me know if you'd like me to take a look as well.

File: _people/fariba-yousefi.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+surname: Yousefi
+othernames: Fariba
+role: Research Software Engineer
+
+---
+
+Fariba Yousefi is in the process of completing her PhD in Machine Learning at the department of Computer Science, University of Sheffield. She recently joined the research software engineering team as a machine learning research engineer at the university of Sheffield.
+
+Her research interests are Bayesian non-parametrics, Gaussian Processes, data scarcity, imbalanced data and multi-task learning. She enjoys working on  healthcare applications, especially medical images.
```
Comment (Robadob) [inline _people/fariba-yousefi.md:14]: "on  healthcare"

Double space? (only visible in the code view, so might be stripped by Jekyyl)

Comment (ptheywood) [reply _people/fariba-yousefi.md:14]: Default css in browser won't render multiple spaces, unless they are in `<pre>` or `<code>` tags (anything with `white-space: pre` or `white-space: pre-wrap` css rules)

**Comment by frb-yousefi** [comment]
Thanks for the review.

File: _people/fariba-yousefi.md (hunk 1 above)
Comment (drj11) [reply _people/fariba-yousefi.md:14]: yeah, it's so minor. Should there be a double space? no. Will it render incorrectly? no. Should it be changed? yes. Would I rather the PR was merged first? yes (reader: it was)

**Pull request merged**